### DISCUSSION TITLE
DBeaver HEX

### DISCUSSION THREAD

**Discussion by gearhead081996** [dbeaver/dbeaver#34366] [DBeaver Development] [answered]
Some of the databases I query have hexadecimal columns .  If I'm using QMF for windows it's easy to convert.  I just use "Select HEX([Column])", pretty easy.  How do I do this in DBeaver?  Is it even possible?  I am using DB2 for z/OS connection.

**Comment by LonwoLonwo** [comment, marked as answer]
Hello @gearhead081996 

Did you try to use DBeaver binary formatter?

![image](https://github.com/dbeaver/dbeaver/assets/45152336/fa8ec9dd-e7f1-4b8a-a16e-0deea5c945d9)

**Reply by gearhead081996** [reply]
Hi @LonwoLonwo, sorry for the extremely late reply to this.  I checked for a while and no replies then I found the email notification in my junk folder I had a reply.  This is what I am looking for.  Not as easy as the other platform where I can just put it in the SQL string, but it will work.  Thank you for the great information!
